releng/lib/check_artifacts.py

```diff
@@ -179,20 +183,26 @@ def do_check_docker(checker: Checker, name: str) -> None:
         if release_channel != '':
             check_tag = f"{check_tag}-{release_channel}"
         for image in images:
-            assert '/ambassador:' in image
+            assert '/emissary:' in image
             check.result = image.split(':', 1)[1]
             assert_eq(check.result, check_tag)
-    subprocess.run(['helm', 'repo', 'rm', 'emissary'], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-    subprocess.run(['helm', 'repo', 'add', 'emissary',
-            'https://s3.amazonaws.com/{}/charts'.format(s3_bucket)], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+
     with checker.check(name="Updating helm repo"):
+        subprocess.run(['helm', 'repo', 'rm', 'emissary'], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+        subprocess.run(['helm', 'repo', 'add', 'emissary',
+                'https://s3.amazonaws.com/{}/charts'.format(s3_bucket)], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+
         run(['helm', 'repo', 'update'])
-    chart_version = ""
-    for line in fileinput.FileInput("charts/emissary-ingress/Chart.yaml"):
-        if line.startswith("version:"):
-            chart_version = line.replace('version:', '').strip()
+
     with checker.check(name="Check Helm Chart"):
+        # chart_version = ""
+        # for line in fileinput.FileInput("charts/emissary-ingress/Chart.yaml"):
+        #     if line.startswith("version:"):
+        #         chart_version = line.replace('version:', '').strip()
+        chart_version = chart_ver
+
```

Comment:
Hmm. I think the right thing here is to just drop setting `chart_version` at all and to replace uses of it with `chart_ver`.

Reply:
Done.